passengerfeedback/passenger_server/main.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	pb "../proto"
+	"context"
+	"google.golang.org/grpc"
+	"log"
+	"net"
+	"strconv"
+)
+
+const (
+	port = ":50051"
+)
+
+var feedbacks []pb.PassengerFeedback
+
+type server struct{}
+
+func (s *server) AddPassengerFeedback(ctx context.Context, in *pb.PassengerFeedbackRequest) (*pb.PassengerFeedbackResponse, error) {
+		for _, fb := range feedbacks {
```

Comment:
- please align indents properly
- using map[string]pb.PassengerFeedback could save you from looping through the whole slice